src/neo/Ledger/Blockchain.cs

```diff
@@ -517,6 +520,17 @@ private void SendRelayResult(IInventory inventory, VerifyResult result)
         private void UpdateCurrentSnapshot()
         {
             Interlocked.Exchange(ref currentSnapshot, GetSnapshot())?.Dispose();
+            extensibleWitnessWhiteList = ImmutableHashSet.Create(
+                NativeContract.NEO.GetCommitteeAddress(currentSnapshot),
```

Comment:
If we plan to move the consensus message to the plugin, we must add all consensus node addresses, not just a multi-signature address.

Reply:
Done